design/FY2017/NFP-SingleDuctAirTerminalConstantVolumeNoReheat.md

```diff
@@ -0,0 +1,212 @@
+
+#### Field: Maximum Air Flow Rate
+
+The design maximum volume flow rate~ (m\(^{3}\)/sec) specified for the AirTerminal:SingleDuct:ConstantVolume:NoReheat object.  This field is autosizable.
+
+#### Field: Design Specification Outdoor Air Object Name
+
+This field allows modifying the behavior of this air terminal so that it is modulated to supply the required outdoor air to the zone.  This field is optional. When the name of an DesignSpecification:OutdoorAir object is entered, the model is changed to adjust the flow rate to provide the volume of outdoor air described by that object.  This feature allows modeling demand controlled ventilation on a zone-by-zone basis using the Outdoor Air Flow per Person rate (specified in the DesignSpecification:OutdoorAir object) and the number of occupants (specified in the People object schedules).  If the outdoor air fraction of the supply air is 1.0, as for a dedicated outdoor air system, the air flow rate will match the outdoor air requirement.  When the outdoor air fraction is less than 1.0, as for a recirculating air system, the terminal air flow will be modulated upward to account for the increased total air flow needed to provide the required flow rate of outdoor air.   The total air flow rate will not exceed the Maximum Air Flow Rate specified above. The volume flow rate is converted to mass flow rate using the standard density of air at Pressure = 101325 Pa, Temperature = 20C, and Humidity Ratio = 0.0.
+
+#### Field: Per Person Ventilation Rate Mode
+
+This field specifies how the outdoor air ventilation rates are calculated when based on a rate per person. It can be either based on the current number of people as affected by time-varying occupancy schedules, or on the constant value for the maximum number of people.  Enter the key CurrentOccupancy for the former and DesignOccupancy for the later.
+
+
+## New idf object based on the proposed IDD object.
+
+#### AirTerminal:SingleDuct:ConstantVolume:NoReheat New Object
+
+```
+AirTerminal:SingleDuct:ConstantVolume:NoReheat,
+
+      NoReheat Zone 1,                !- Name of System
+      AlwaysOnFanAvailSched,          !- Availability Schedule Name
+      Zone 1 Unit Air Outlet Node,    !- Air Outlet Node Name
+      Zone 1 Unit Air Inlet Node,     !- Air Inlet Node Name
+      0.60,                           !- Maximum Air Flow Rate {m3/s}
+      ,                               !- Design Specification Outdoor Air Object Name
+      ;                               !- Per Person Ventilation Rate Mode
+
+
+```
+
+## Outputs Description ##
+
+  HVAC,Sum,Zone Air Terminal Sensible Heating Energy {[}J{]}
+
+  HVAC,Sum,Zone Air Terminal Sensible Cooling Energy {[}J{]}
+
+  HVAC,Average,Zone Air Terminal Sensible Heating Rate {[}W{]}
+
+  HVAC,Average,Zone Air Terminal Sensible Cooling Rate {[}W{]}
```

Comment:
Interesting, the uncontrolled terminal unit is the only one with these outputs, so we need to keep these for compatibility.  The VAV terminal units that have a design specification outdoor air field, also have and output for "Zone Air Terminal Outdoor Air Volume Flow Rate" - it would be good to include that here also.

Reply:
@mjwitte We have the same variables name reported for all zone equipment and air terminal at ZoneTempPredictorCorrector.cc. From what I recall under some circumstances these two variables may have different values. We have seen this issue in the past.

 I will add the "Zone Air Terminal Outdoor Air Volume Flow Rate" output variable.